### PR TITLE
ECF-RP-129: Remove old fields from schools & import data in background job

### PR DESCRIPTION
**Pull request by ethanmills**
### Context
PR 2/3 for this ticket. In this one I clean up old db columns and finish up the work of getting the school import to run as a background job

### Changes proposed in this pull request
- remove obsolete db columns
- Make importing school data part of the `db:seed` task, but run it in the background so deployment doesn't take an eternity

### Testing
Removal of columns is covered by unit tests.
Import of data should be evident from review app 🤞

### CODE REVIEW

**Comment by ethanmills** [comment]
[Generate fixture from Sparsity data](https://trello.com/c/X8OpA38K/129-generate-fixture-from-sparsity-data)

**Comment by ethanmills** [comment]
The import is currently failing with 
```
uninitialized constant SchoolDataImporter::Addressable  
/app/app/services/school_data_importer.rb:58:in `row_to_school'
```

File: Gemfile [hunk 1]
```diff
@@ -50,13 +50,16 @@ gem "httpclient"
...
 # Database based asynchronous priority queue system
+gem "daemons"
 gem "delayed_cron_job"
 gem "delayed_job_active_record"
 
 platform :mswin, :mingw, :x64_mingw do
   gem "wdm", ">= 0.1.0"
 end
 
+gem "addressable"
```
Comment (amazimbe) [inline Gemfile:61]: Just wondering what we are using this for?

File: Gemfile [hunk 2]
```diff
@@ -50,13 +50,16 @@ gem "httpclient"
 gem "view_component", require: "view_component/engine"
 
 # Database based asynchronous priority queue system
+gem "daemons"
```
Comment (amazimbe) [inline Gemfile:53]: What are we using this for?

File: Gemfile (hunk 1 above)
Comment (ethanmills) [reply Gemfile:61]: It's used in the [school import](https://github.com/DFE-Digital/early-careers-framework/blob/5f7509105b5b88e503e5d3d92cfd6a33fc294648/app/services/school_data_importer.rb#L58). I have no idea why it stopped working, but when I was trying to run the import it complained about the lack of Addressable. Perhaps we removed something that had it as a transient dependency?

File: Gemfile (hunk 2 above)
Comment (ethanmills) [reply Gemfile:53]: To [run delayed jobs](https://github.com/collectiveidea/delayed_job#running-jobs)

**Review by rupollbabu — APPROVED** [review]
No impact on test after removal of old fields and import data, Will go through complete test after completion of rest of dependency tickets(Ticket 130).

**Comment by amazimbe** [comment]
> The import is currently failing with
> 
> ```
> uninitialized constant SchoolDataImporter::Addressable  
> /app/app/services/school_data_importer.rb:58:in `row_to_school'
> ```
> 
> I'm looking into this

@ethanmills did you manage to resolve this issue?

**Comment by ethanmills** [comment]
> > The import is currently failing with
> > ```
> > uninitialized constant SchoolDataImporter::Addressable  
> > /app/app/services/school_data_importer.rb:58:in `row_to_school'
> > ```
> > 
> > 
> > I'm looking into this
> 
> @ethanmills did you manage to resolve this issue?

Yep, that's what Addressable was for https://github.com/DFE-Digital/early-careers-framework/pull/97#discussion_r576647716

**Review by amazimbe — APPROVED** [review]

File: terraform/workspace-variables/review.tfvars [hunk 3]
```diff
@@ -10,4 +10,4 @@ paas_app_stopped = false
 paas_web_app_deployment_strategy = "blue-green-v2"
 paas_web_app_instances = 1
 paas_web_app_memory = 512
-paas_web_app_start_command = "bundle exec rake cf:on_first_instance db:migrate && rails db:seed && rails s"
+paas_web_app_start_command = "bundle exec rake cf:on_first_instance db:migrate && rails db:seed && /app/bin/delayed_job start && rails s"
```
Comment (kicferk1) [inline terraform/workspace-variables/review.tfvars:13]: Do we need this change?

So far as I see it, we fire off the importer from our seed setup, so I would be surprised if we need to fire it off as a separate command too.

And if we need it for any delayed job to run, hmm, maybe we want it on start of every application, not just review apps.

Comment (ethanmills) [reply terraform/workspace-variables/review.tfvars:13]: It's the daemon task that runs delayed jobs. In other environments I was wondering if perhaps we wanted it as a different worker instance. Was leaving that for future PR

Comment (kicferk1) [reply terraform/workspace-variables/review.tfvars:13]: Sure, can be a future PR, so long as we have a trello card for it so it's not forgotten :)

**Pull request merged**